homeassistant/components/arest/sensor.py

```diff
@@ -162,7 +162,7 @@ def __init__(
     def update(self) -> None:
         """Get the latest data from aREST API."""
         self.arest.update()
-        self._attr_available = self.arest.available
+        self._attr_available = self.arest._attr_available
```

Comment:
This fixes the bug but it uses wrong code style. Much better is to keep the old name and revert the name in `class ArestData` to `available`. The reason for that is that names starting with underline are used for private data but the available variable is public (you access it from a different class).

So here you should keep the current code `self._attr_available = self.arest.available` and in the class `ArestData` you should revert the variable to be named `available`.

Reply:
yes, I will change.